spec/request/api/v1/topic/index_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+describe 'GET api/v1/topics', type: :request do
+  subject do
+    get api_v1_topics_path, headers: auth_headers, as: :json
+    response
+  end
+
+  let(:filename) { 'generic-logo.png' }
+  let(:file) do
+    {
+      io: File.open(
+        File.expand_path(File.join(File.dirname(__FILE__) + '/../../../..', 'fixtures', filename))
+      ),
+      filename: filename
+    }
+  end
```

Comment:
Maybe we can create a trait on the factory so we always create a topic with the same image